var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -49,15 +50,22 @@ class Adios(AutotoolsPackage):
             description='Enable Fortran bindings support')
 
     variant('mpi', default=True, description='Enable MPI support')
+    variant('serial', default=False, description='Build serial version by fully disabling MPI support')
     variant('infiniband', default=False, description='Enable infiniband support')
+    variant('mxml', default=False, description='Build with external mxml')
```

Comment:
Are you saying that if you choose `~mpi` and `~mxml` it will build with its own internal copy of those libraries? We don't normally like doing that in Spack because it prevents you from controlling what gets installed. When I see a package with a `+mpi` variant, I automatically assume that `+mpi` means "with mpi" and `~mpi` means "without mpi". If this package doesn't conform to that standard, it might trip other people up.